spec/models/event_spec.rb

```diff
@@ -45,21 +45,69 @@
 
     it "should not be possible to have an enddate, that is before the startdate" do
       expect(event).to be_valid
-      event.enddate = Date.new(2017,11,30)
+      event.enddate = Date.today
       expect(event).not_to be_valid
     end
 
     it "should be possible to get the duration in day of an event" do
-      expect(event.duration).to eq(5)
+      expect(event.duration).to eq(2)
     end
 
     it "should only show active events" do
       @new_event = FactoryBot.create(:event, deadline: Date.current)
-      @old_event = FactoryBot.create(:event)
+      @old_event = FactoryBot.create(:event, deadline: Date.yesterday)
 
       expect(Event.active).to include(@new_event)
-      expect(Event.active).to_not include(@old_event)
+      expect(Event.active).not_to include(@old_event)
       expect(Event.all).to include(@new_event, @old_event)
     end
 
+it "should have an attribute deadline" do
+    @date = Date.tomorrow
```

Comment:
If you don't plan on using variables across tests there is no need to for making them instance variables. Just delete the @ :)